dll/win32/browseui/shellfind/CFindFolder.cpp

```diff
@@ -272,7 +273,9 @@ static UINT RecursiveFind(LPCWSTR lpPath, _SearchData *pSearchData)
             uTotalFound += RecursiveFind(szPath, pSearchData);
```

Comment:
L263/264 (where I can't comment):
Should this new checkbox affect whether hidden directories show up in search results, and whether hidden directories get recursively searched at all?

The expression there also looks similar to the `else if` expression below, which may be a good indication that a potential helper function (see below) can be re-used here.

Reply:
@ThFabba, as you suggested, I was able to use the helper functions in both places. Thanks.